plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -254,14 +254,26 @@ private function add_general_group_blocks() {
 				),
 			)
 		);
-		$description_section->add_block(
+		$description_field_block = $description_section->add_block(
 			array(
 				'id'        => 'product-description',
 				'blockName' => 'woocommerce/product-description-field',
 				'order'     => 10,
 			)
 		);
 
+		$description_field_block->add_block(
+			array(
+				'id'        => 'product-description__content',
+				'blockName' => 'woocommerce/product-summary-field',
```

Comment:
Hmm this is an interesting approach, using the summary block as an inner block instead of re-using some of the same summary block components.
I am not opposed to this, but is there any specific benefits of doing it this way?

Reply:
Usability/extensibility. Reusing the block is the more efficient way to reuse resources. We could reuse the components instead, but in the end, we will create a block that will almost a clone the summary one.

By the way, if we take this path, we should probably create a `<ParagraphControlBlock />`, or something like that usable everywhere we want to implement a textarea with RichText support